PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+### Requirements
```

Comment:
Not completely sure I'm having a good grasp about this, but should this be in root folder and not under .github?

Reply:
Github documentation is not completely clear about this, we can try as it is now and if is not working move the PR Template to the root.